Updated pdp to figma

george-icarcover opened the pull request.

devwilliamy commented on src/app/(main)/car-covers/components/CarPDP.tsx:
```diff
@@ -64,7 +64,7 @@ export default function CarPDP({
   return (
     <>
-      <CarCoverSelector
+      <MobileCarCoverSelector
```
What was the point of changing this to `MobileCarCoverSelector`? 
If there's no reason, please change it back to `CarCoverSelector`.

devwilliamy commented on src/app/(main)/car-covers/components/CustomerImagesDrawer.tsx:
```diff
@@ -0,0 +1,182 @@
+            <TabsList
+              className="flex flex-col bg-white pt-[24px]"
+              onChange={(e) => {
+                console.log('Current Tab Index:' + e);
```
Please remove console logs when you're done with it
You can also remove this onChange? Doesn't look like it's doing anything.

devwilliamy commented on src/app/(main)/car-covers/components/CustomerImagesDrawer.tsx:
```diff
@@ -0,0 +1,182 @@
+                {[
+                  { title: 'Customer Images', value: 'customer-images' },
+                  { title: 'Customer Reviews', value: 'customer-reviews' },
+                ].map(({ title, value }, index) => (
```
Extract this array outside of the component itself.

devwilliamy commented on src/app/(main)/car-covers/components/CustomerImagesDrawer.tsx:
```diff
@@ -0,0 +1,182 @@
+              className="border-t-2 border-[#C8C7C7]"
+            >
+              <div className="mx-auto flex max-h-[76vh] min-h-[76vh] w-full flex-col px-4 pt-[29px]">
+                <div className="mx- grid grid-cols-3 gap-2 overflow-y-auto">
```
What is this mx-? Please delete if it's not needed

devwilliamy commented on src/app/(main)/car-covers/components/CustomerImagesDrawer.tsx:
```diff
@@ -0,0 +1,182 @@
+            >
+              <div className="mx-auto flex max-h-[76vh] min-h-[76vh] w-full flex-col px-4 pt-[29px]">
+                <div className="mx- grid grid-cols-3 gap-2 overflow-y-auto">
+                  {[...Array(25)].map((_, index) => (
```
Please find out what these images are supposed to be or confirm if Calvin is okay with these being dummy images.

george-icarcover replied:
Calvin wants us to remove it until we have the pictures.

devwilliamy commented on src/app/(main)/car-covers/components/CustomerImagesDrawer.tsx:
```diff
@@ -0,0 +1,182 @@
+                <Carousel setApi={setCustomerCarouselApi}>
+                  <CarouselContent>
+                    {reviewData &&
+                      reviewData.map((review, index) => (
```
Are you able to just do `reviewData?` here? If you need the `reviewData &&` can leave as is

devwilliamy commented on src/app/(main)/car-covers/components/MobileCarCoverSelector.tsx:
```diff
@@ -21,7 +21,7 @@ const EditVehiclePopover = dynamicImport(
 );
 
-export function CarCoverSelector({
+export function MobileCarCoverSelector({
```
Please rename back to `CarCoverSelector `if there's no need for this to be named `MobileCarCoverSelector`

devwilliamy commented on src/app/(main)/car-covers/components/MobileImageCarousel.tsx:
```diff
@@ -71,30 +87,62 @@ export const MobileImageCarousel = ({
-          ))}
-          
+          {productImages.map((image, index) => {
+            if (index <= 1) return;
```
What is this doing?

george-icarcover replied:
There are 2 extra items in the carousel that aren't in the product images map.
- This prevents the first two items in product images map from showing.

devwilliamy commented on src/app/(main)/car-covers/components/ProductContent.tsx:
```diff
@@ -67,12 +90,39 @@ export function ProductContent({
+    useState<boolean>(false);
   const router = useRouter();
+
+  // const [customerReviewsIndex, setCustomerReviewsIndex] = useState<number>(0);
```
If you're not using these, please delete the comments

devwilliamy commented on src/app/(main)/car-covers/components/ProductContent.tsx:
```diff
@@ -293,6 +366,24 @@ export function ProductContent({
+        setCustomerImagesIndex={setCustomerImagesIndex}
+      />
+      <span className="mb-[32px] flex w-full gap-[7px] overflow-x-auto">
+        {[...Array(12)].map((_, index) => (
```
Please confirm what images to use here

devwilliamy commented on src/components/PDP/ProductChecklist.tsx:
```diff
@@ -51,7 +51,7 @@ export function ProductChecklist() {
             <div className="ml-2 md:ml-4">
               <p className="text-base font-normal text-white md:text-lg lg:text-xl">
-                Cars frequently exposed to <b>rain</b> and <b>humidity.</b>
+                Cars frequently exposed to <b>rain and humidity.</b>
```
it's better to use `<span className=""/>`  instead of `<b>`, but if the font weight is the same then it's fine

devwilliamy reviewed (changes requested):
I can't comment on it, but the icons should be in `/components/icons` and not in the `public` folder.

Please checkout the comments, will be doing a UI check tmrw.

devwilliamy commented:
# Header:

- sale banner should be right above hero banner
- You're missing the profile icon
- The hamburger icon does nothing and should only show on mobile
- Either ask for how the hamburger drop down should work, or just replace it with a profile icon becuase we don't have anything else to have a need for it
- the search bar isn't aligned with thte logo and the icons

Yours:
![image](https://github.com/femi-icarcover/coverland/assets/156253900/54eed208-8a82-4c8a-ae11-e5586c06130a)

Figma:
![image](https://github.com/femi-icarcover/coverland/assets/156253900/3c850037-a71a-4d23-963a-82926ad27246)

# PDP

More padding between the  border and the title

![image](https://github.com/femi-icarcover/coverland/assets/156253900/5ee7edb7-9ab8-452e-afd5-87d049c0b9e3)


There are some font size differences here. Please check the Figma and match.
![image](https://github.com/femi-icarcover/coverland/assets/156253900/a5e291be-b26e-42ce-87f3-43d3ffc07cf9)

- Please confirm if you need a period at the end
- Please confirm if we should have the learn more. If yes, what should it do? 

![image](https://github.com/femi-icarcover/coverland/assets/156253900/8c9759eb-d895-40a5-ba7f-bcddab0ea2c5)

- Please update these with the actual pictures. We should **not** release this with example photos

![image](https://github.com/femi-icarcover/coverland/assets/156253900/e8181b28-aae1-4c7e-a4e7-bbc953375534)

- More padding below the agent 

![image](https://github.com/femi-icarcover/coverland/assets/156253900/df77e73d-fce6-4cc8-b52f-1b9ada6cae5b)

- This needs to be updated

![image](https://github.com/femi-icarcover/coverland/assets/156253900/5eb72168-d9c3-4983-887a-7ea53a68560e)

- ENgineered to perfection (24px -> 22px) but don' really care about this one. Looks fine as is.

![image](https://github.com/femi-icarcover/coverland/assets/156253900/7c5bc30d-32c1-4a50-9495-13d4b07267b3)

- There's extra padding here, making the space bigger than expected. Your margin is correct but there's more space due to the padding. Please adjust the margin w/ the padding or adjust the padding

![image](https://github.com/femi-icarcover/coverland/assets/156253900/00a7f185-7e46-4574-986e-c72cbd141db5)

- Update line height to 24px and the ones below too. I'm not going to SS them all

![image](https://github.com/femi-icarcover/coverland/assets/156253900/ba769527-f19c-4ac7-9fdc-b252887f17f7)

- More padding between picture and title (and also the line height)

![image](https://github.com/femi-icarcover/coverland/assets/156253900/994cb7f0-b178-481b-865e-0cd4e8d7fe61)

- Please confirm with Ashley or Calvin about the capitalization at the beginning

![image](https://github.com/femi-icarcover/coverland/assets/156253900/757911b1-4545-43e0-886c-331ce0fc65f5)

- Increase left padding of text to left edge of photo. Decrease padding between checkmark and the text. Make sure if the text is two lines, the checkmark should line up at the beginning

![image](https://github.com/femi-icarcover/coverland/assets/156253900/adbf3191-19c2-4867-ba90-14efcf7a8bf9)

- Increase left padding of text to left edge of photo

![image](https://github.com/femi-icarcover/coverland/assets/156253900/4e7f58d0-1dbc-4128-8dd0-2b7e0c5fb75b)

- Update the text content font size (16px->14px) (eg. "Enjoy free gound shipping...")
- Upadding bottom padding below Return details text content 

![image](https://github.com/femi-icarcover/coverland/assets/156253900/a8ed6093-bb99-4aa8-adef-438b2a67f623)


- I think we've talked about it before to leave it at 100 but I checked again and 400 looks ok here. Update to font weight 400

![image](https://github.com/femi-icarcover/coverland/assets/156253900/2ff72b6d-9a2d-4363-919a-a374bbf58943)

- Increase padding between the border line and the 4.9.
- Confirm if you're supposed to add review images here yet

![image](https://github.com/femi-icarcover/coverland/assets/156253900/5761889f-d5cb-474d-8850-9e0a44543452)

devwilliamy commented:
Also the notification should be next to the cart like how it is on mobile

![image](https://github.com/femi-icarcover/coverland/assets/156253900/6fa12883-c318-4928-a40e-7d3b5d4a9547)

george-icarcover commented:
** Confirmed, replaced with profile for the time bieng**
- Asked Ashley about what to do with the hamburger
** Confirmed, added capitals**
- Please confirm with Ashley or Calvin about the capitalization at the beginning
** Confirmed and removed ** 
- Please confirm if you need a period at the end
- Please confirm if we should have the learn more. If yes, what should it do?
- Confirm if you're supposed to add review images here yet
- Confirm if this is supposed to be lower cased (probably not)
** Confirmed working on all cover-type pages**
